contracts/CAT_FACTORY.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
```

Comment:
Let's rename the file to "CATFactory.sol".

And let's change the license to AEL.